src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Modal.css';
+import { ModalType } from 'canvas/Enums';
+
+interface ModalProps {
+    modal: Modal;
+}
```

Comment:
missing semi

Reply:
I don't see where the semicolon is missing here.

Reply:
my bad it's not. oops.